store/tikv/client_batch.go

```diff
@@ -224,14 +225,16 @@ func (c *batchCommandsClient) reCreateStreamingClient(err error) bool {
 			zap.String("target", c.target),
 		)
 		c.client = streamClient
-		return true
+		return err
+	}
+	if !skipLog {
+		logutil.BgLogger().Error(
+			"batchRecvLoop re-create streaming fail",
+			zap.String("target", c.target),
+			zap.Error(err),
```

Comment:
I prefer to keep this log.
When we operating a long-live service, it'd be better to have logs reflecting the current state of it.

Reply:
Have you ever seen the error log from our user? I bet you'll withdraw this comment if you see it.